### DISCUSSION TITLE
Website not able to load

### DISCUSSION THREAD

**Discussion by Abhi132004** [sanidhyy/mern-admin#20] [General]
I am trying to load this website , but its dashboard and charts are not responding

**Comment by sanidhyy** [comment]
The site server was deployed on heroku which is no longer free. That's the reason it's no longer working. You can deploy it yourself to Heroku or any other alternatives (like render.com). They do offer free trial/credits.

**Comment by AbhiData132004** [comment]
I have also tried to run it on my localhost,
There is some kind of error regarding the loading of components, I have connected the server properly using mongo db and database were also created , but it's still loading till death, can you please assist me over this issue
![image](https://github.com/user-attachments/assets/1feb1a7e-5505-4e4b-a4b6-b199f2aff592)

**Reply by sanidhyy** [reply]
Can u share your console screenshot when u are getting error?

**Reply by AbhiData132004** [reply]
no error in backend as well as in client, so the console is not showing any error

**Reply by sanidhyy** [reply]
Can u check network tab in your browser if there is any 500 error or 404 error response?
